sdk/client_test.go

```diff
@@ -8,15 +8,19 @@ import (
 )
 
 const (
-	authHeader = "Authorization"
-	authPrefix = "Bearer "
+	authHeader    = "Authorization"
+	authPrefix    = "Bearer "
+	eTagHeader    = "ETag"
+	ifMatchHeader = "If-Match"
 )
 
 func TestHeaders_Add(t *testing.T) {
 	t.Parallel()
 
 	Convey("Given the sdk Headers struct contains a value for ETag", t, func() {
-		req := &http.Request{}
+		req := &http.Request{
```

Comment:
can this be moved at the top of the test to avoid repetition of code?

Reply:
good idea, done